public/download.php

```diff
@@ -113,6 +114,15 @@
                 </span>
             </div>
 
+            <br/>
+            <h2 class='longheader' id='rapplewin'>Apple II Emulator</h2>
+
+            <div class='largeicon'>
+                <span class='clickablebutton'>
+                    <a href="/bin/RAppleWin.zip">Download RAppleWin v<?php echo $latestRAQUASI88Ver; ?> for Windows</a>
```

Comment:
this should be `$latestRAppleWinVer`, as well as the reference on line 13.

Reply:
Fixed this page on my end.